ros_ign_gazebo_demos/launch/image_bridge.launch.py

```diff
@@ -23,41 +23,40 @@
 from launch.substitutions import LaunchConfiguration
 from launch_ros.actions import Node
 
+
 def generate_launch_description():
 
     pkg_ros_ign_gazebo_demos = get_package_share_directory('ros_ign_gazebo_demos')
+    pkg_ros_ign_gazebo = get_package_share_directory('ros_ign_gazebo')
 
-    # Ignition Gazebo
     ign_gazebo = IncludeLaunchDescription(
         PythonLaunchDescriptionSource(
-            os.path.join(pkg_ros_ign_gazebo_demos, 'launch', 'ign_gazebo.launch.py'),
-        )
+            os.path.join(pkg_ros_ign_gazebo, 'launch', 'ign_gazebo.launch.py')),
+        launch_arguments={
+            'ign_args': '-r sensors_demo.sdf'
+        }.items(),
     )
 
     # RQt
     rqt = Node(
         package='rqt_image_view',
-        node_executable='rqt_image_view',
+        executable='rqt_image_view',
         arguments=['/camera'],
         condition=IfCondition(LaunchConfiguration('rqt'))
     )
 
     # Bridge
     bridge = Node(
         package='ros_ign_image',
-        node_executable='image_bridge',
+        executable='image_bridge',
```

Comment:
This is only [changing from Foxy](https://index.ros.org/doc/ros2/Releases/Release-Foxy-Fitzroy/#launch-ros), so we can't land it on the `dashing` branch.